packages/vscode-extension/src/common/Project.ts

```diff
@@ -8,6 +8,7 @@ export type DeviceSettings = {
     longitude: number;
     isDisabled: boolean;
   };
+  biometricEnrollment: boolean;
```

Comment:
Maybe `biometricsRegistered`, `hasEnrolledBiometrics`, etc?

Reply:
BTW, how do we sync device settings across workspaces? I didn't see code reading things using xcrun.

Reply:
we don't sync across workspaces, we remember workspace settings tho, by calling changeSettings on boot